### ISSUE TITLE
[Bug] Provide a short description of the bug here

### ISSUE THREAD

**Issue by MatzeMuc86** [jlesage/docker-jdownloader-2#239]
### Current Behavior

Files are downloaded properly but can not be seen by same user until rights are re-written on host

### Expected Behavior

Files can be seen by user specified by user-id on the ost

### Steps To Reproduce

Download any file and try to see the file within host system

### Environment

Synology DS 7.2

### Container creation

`
services:
  jdownloader:  
    container_name: jdownloader
    image: jlesage/jdownloader-2:latest
    hostname: jdownloader
    domainname: xxx
    #user: 1027:100
    ports:
      - "5800:5800"
      - "3129:3129"
    #  - "5900:5900"
    environment:
      TZ: "Europe/Berlin"
      USER_ID: 1027
      GROUP_ID: 100
      KEEP_APP_RUNNING: 1
      #SUP_GROUP_IDS: 666
      MYJD_USER: xx
      MYJD_PASSWORD: xx
      MYJD_DEVICE_NAME: JDownloader@Docker
      UMASK: "0002"
    dns:
      #- 192.168.186.2
      - 9.9.9.9
    volumes:
      - /volume3/docker/jdownloader/config:/config:rw
      - /volume1/Downloads/JDownloader:/output:rw
      #- /volume3/Downloads:/output:rw
    restart: always
`

### Container log

```text
-
```

**Comment by jlesage** [comment]
Files created by the container should be owned by user ID 1027 and group ID 100. Is this the case ?

**Comment by MatzeMuc86** [comment]
You are right - issue must be on some other app using the files later and not seeing them. Thank you!
